project/ScalafixBuild.scala

```diff
@@ -58,7 +58,10 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
           val xsource3 = TargetAxis(sv, xsource3 = true)
 
           (prevVersions :+ xsource3).map((sv, _))
-        } :+ (scala3Next, TargetAxis(scala213))
+        } ++ Seq(
+          (scala3Next, TargetAxis(scala213)),
+          (scala3Next, TargetAxis(scala3LTS))
```

Comment:
Next vs LTS should not happen in practice because [interface should load the right version](https://github.com/scalacenter/scalafix/blob/590db62f1984848a2765a834865cc96b3f8c83ed/scalafix-interfaces/src/main/java/scalafix/interfaces/Scalafix.java#L128-L134), but this is an opportunity to test dynamic loading

Reply:
The problem is that this means enabling the flag for all cases, including the ones where it's not needed. So effectively not testing the regular path. I'll test dynamic loading with an adhoc, non-expect test I guess.